### DISCUSSION TITLE
"Do not hyphenate headings" option

### DISCUSSION THREAD

**Discussion by ichnilatis-gr** [koreader/koreader#10739] [Q&A] [answered]
I can't find an option in "Style tweaks" that prevents hyphenation in headings. Is there such an option? If not, what should be the code for such at tweak?

**Comment by Frenzie** [comment, marked as answer]
The one that's there already does that, but perhaps the name is mildly deceptive.
https://github.com/koreader/koreader/blob/47dae6c5dd11608c02e478f1cd643a908b449a98/frontend/ui/data/css_tweaks.lua#L346-L352

**Reply by ichnilatis-gr** [reply]
Indeed. We should read the description in the pop-up window and keep it in mind... :-)
Thanks!
